frontend/app/modules/sample/sample-module.js

```diff
@@ -15,6 +15,15 @@
           templateUrl: 'sample-2.html',
           controller: 'SampleController',
           controllerAs: 'vm'
+      }).state('chat-app.landing', {
+        url: '/landing',
+        templateUrl: 'landingPage.html',
+        controller: 'SampleController',
+        controllerAs: 'vm'
+      }).state('chat-app', {
```

Comment:
The name of this state conflicts with the root state defined in `app.js`. This state is actually not needed - you can get rid of it safely.